src/main/resources/scripts/-examples/functions.sk

```diff
@@ -34,22 +34,22 @@ command /appleexample:
 # Please note that self-calling functions can loop infinitely, so use with caution.
 #
 
-function destroyOre(source: block) :: blocks:
+function destroyGold(source: block) :: blocks:
     add {_source} to {_found::*}
     break {_source} naturally using an iron pickaxe
     loop blocks in radius 1 of {_source}:
-        loop-block is any ore
+        loop-block is tagged as minecraft block tag "gold_ores"
```

Comment:
I think we should change the examples that now have to use tags because they're kind of clumsy and not great as introductions.

Reply:
i don't think it hurts to have an example of tags, but maybe we'd want a specific example script for tags instead

Reply:
Yes I agree about a specific example for tags, but what I don't like is that most of the examples are built with ultimate simplicity in mind, everything's in very clear and simple language using basic features that a new reader can understand, whereas this presupposes understanding what a material tag is (plus it's kind of nasty syntax now that I look at it)

Reply:
We could change it to just be like `gold ore or deepslate gold ore` if you'd like

Reply:
Some change is important, or declaring an alias is fine.